README.md

```diff
@@ -317,6 +317,53 @@ The full source code is available [here](./examples/catcma.py).
 </details>
 
 
+#### Maximum a Posteriori CMA-ES [Hamano et al. 2024b]
+MAP-CMA is a method that is introduced to interpret the rank-one update in the CMA-ES from the perspective of the natural gradient.
+The rank-one update derived from the natural gradient perspective is extensible, and an additional term, called momentum update, appears in the update of the mean vector.
+
+<details>
+
+<summary>Source code</summary>
+
+```python
+import numpy as np
+from cmaes import MAPCMA
+
+
+def rosenbrock(x):
+    dim = len(x)
+    if dim < 2:
+        raise ValueError("dimension must be greater one")
+    return sum(100 * (x[:-1] ** 2 - x[1:]) ** 2 + (x[:-1] - 1) ** 2)
+
+
+if __name__ == "__main__":
+    dim = 20
+    optimizer = MAPCMA(mean=3 * np.ones(dim), sigma=2.0, momentum_r=dim)
```

Comment:
It's strange that the optimization on the Rosenbrock function starts from m=3. Please adjust it to m=0.0 and sigma=0.5. The sigma setting is to avoid local optima and make the check easier.